data/wanderers start.txt

```diff
@@ -2439,7 +2439,13 @@ mission "Wanderers Ap'arak 3: Pug"
 		government "Pug"
 		ship "Pug Arfecta" "Hork Ekekek Ptui"
 		ship "Pug Arfecta" "Graak Sput Kchoo"
+		
+	on complete
+		"reputation: Wanderer" <?= -1
+		dialog
+			`After destroying the Pug ships, a Wanderer ship hails you. " We have warned you against [attacking, angering] the Pug. This behavior is [unacceptable, untolerable]."`
```

Comment:
Maybe change "untolerable" (which isn't a word) to "unforgivable", which might make more sense given that they then immediately become hostile.

Reply:
They probably meant 'intolerable' https://www.merriam-webster.com/dictionary/intolerable but unforgivable is a good synonym too

Reply:
Fixed